scene/2d/physics_body_2d.h

```diff
@@ -347,10 +358,15 @@ class CharacterBody2D : public PhysicsBody2D {
 	Vector2 up_direction = Vector2(0.0, -1.0);
```

Comment:
Also `platform_layer` is missing initialization value, might as well fix it while doing adjustments :)